include/maths/CBoostedTreeImpl.h

```diff
@@ -735,7 +736,7 @@ class MATHS_EXPORT CBoostedTreeImpl final {
     //!
     //! \note This number will only be used if the regularised loss says its
     //! a good idea.
-    std::size_t maximumTreeSize(const core::CDataFrame& frame) const;
+    std::size_t maximumTreeSize(const core::CPackedBitVector& frame) const;
```

Comment:
nit: the parameter seems not to be a (data) frame anymore, impl uses `trainingRowMask`

Reply:
[bf51751](https://github.com/elastic/ml-cpp/pull/640/commits/bf51751decfd8c7f8aab367652fb87c07bd29667)